### DISCUSSION TITLE
Requesting at least 1 label for query pattern change in the Loki query builder mode

### DISCUSSION THREAD

**Discussion by SanthoshS20** [grafana/grafana#53834] [Observability]
**What happened:**
In Loki query dashboard, change in the query pattern for a query field results in warnings **_You need to specify at least 1 label filter_**. I created a new query field in the Loki dashboard. In the Query builder mode, I changed the query pattern. Suddenly, a warning popup message was shown. I don't see this issue in the query code mode.

**What you expected to happen:**
In the Loki query dashboard, If a query field doesn't have a label, it should not show any warning if we did a change in the query pattern. 

**How to reproduce it (as minimally and precisely as possible):**
Step 1: Go the the Loki dashboard.
Step 2: Switch to the Loki builder mode.
Step 3: Change the query pattern for the query field which was present. It will show the warning message.

**Anything else we need to know?:**
![image](https://user-images.githubusercontent.com/37366893/185149717-71e5b286-9e5a-4a8a-bffe-9f89fee7e73c.png)
_**No Warning was present for changing the query patter in the code mode. See the below image for reference.**_
![image](https://user-images.githubusercontent.com/37366893/185149888-d39bbea4-0dc6-48f3-9a2b-e2915a641a7b.png)

**Environment:**
Grafana version: 9.0.7
User OS & Browser: Windows 10 & Chrome

**Comment by ivanahuckova** [comment]
In builder, this is a correct behaviour as query without label filter is invalid.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
